packages/core/src/rules/chain_mainly_declarations.ts

```diff
@@ -4,6 +4,10 @@ import {BasicRuleConfig} from "./_basic_rule_config";
 import * as Statements from "../abap/2_statements/statements";
 import {IRuleMetadata, RuleTag} from "./_irule";
 import {ABAPFile} from "../abap/abap_file";
+import {EditHelper, IEdit} from "../edit_helper";
+import {StatementNode} from "../abap/nodes";
+import {Position} from "..";
```

Comment:
```suggestion
import {Position} from "../position";
```

importing the index somtimes messes stuff up